07-js_html_css_dom/portfolio/index.js

```diff
@@ -0,0 +1,36 @@
+document.querySelector('#dark-mode-button').addEventListener('click', darkMode)
+
+function darkMode() {
+    let body = document.body;
+    let  footer = document.querySelector('#footer');
+    let darkModeButton = document.querySelector('#dark-mode-button');
+    if (body.classList.contains('dark-mode')) {
+
+        body.classList.remove('dark-mode')
+        footer.classList.remove('dark-mode');
+        darkModeButton.innerText = 'dark mode';
+    } else {
+        body.classList.add('dark-mode')
+        footer.classList.add('dark-mode');
+        darkModeButton.innerText = 'light mode';
+    }
+}
+
+document.querySelector('#text-size-smaller-button').addEventListener('click', decreaseBaseTextSize)
+document.querySelector('#text-size-bigger-button').addEventListener('click', increaseBaseTextSize)
+
+function decreaseBaseTextSize() {
+    let textSize = document.querySelector('#text-size');
+    textSize.innerText = parseInt(textSize.innerText) - 1;
+    let currentFontSize = window.getComputedStyle(document.body).fontSize;
```

Comment:
Thanks. So this says: if the body's fontSize is empty (type safe) then let the currentFontSize be equal to 16, otherwise use the body's fontSize. So basically, set the default to 16.